javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -53,51 +54,15 @@ import com.github.javaparser.ast.type.*;
 /**
  * <p>This class was generated automatically by javacc, do not edit.</p>
  */
-final class ASTParser {
+public class ASTParser extends ASTParserBase {
+	// Make the parser public for now
```

Comment:
I explained in the commit message that this is temporary (because of the package move and the need to be visible by the JavaParser class).

I plan to make a further front-end in beetween the JavaParser class. It will provide better error handling.